src/core/ThemeProvider/ThemeProvider.tsx

```diff
@@ -14,15 +14,21 @@ export type ThemeProviderProps = {
    * Optional children.
    */
   children?: React.ReactNode;
+  /**
+   * Document to which the theme will be applied.
+   * Can be specified to handle popup windows.
+   * @default document
+   */
+  ownerDocument?: Document;
```

Comment:
We were thinking about it, safer option might be to have a prop called `themeOptions`, which could be an object with `ownerDocument` property for now. So we have an easy way to add more stuff in future, when needed.
For now that might be enough and we would need to think how we gonna handle this in our components, when needed.

Reply:
ThemeOptions implemented,

Reply:
Shouldn't it be in `useTheme` (where `ThemeType` is also defined)?

Reply:
Moved